test/unit/data/ScoutTitles.json

```diff
@@ -14,21 +14,6 @@
       "image_url":
         "https://static01.nyt.com/images/2018/06/21/business/21Techfix/merlin_139815324_5d0f3563-0071-4a44-926f-3841b7e7fb44-facebookJumbo.jpg"
     },
-    {
-      "item_id": "2229373616",
-      "sort_id": 1,
-      "resolved_url":
-        "https://medium.com/s/trustissues/blockchain-disciples-have-a-new-goal-running-our-next-election-2bfe4dff3c7",
-      "title":
```

Comment:
I have an email in to figure out what this is not returning correct value of article